docs/source/community/content-community.rst

```diff
@@ -2,6 +2,49 @@
 Community Guides
 ================
 
+
+Welcome to the Community Guide for Jupyter. This section try to introduce many
+of the Jupyter-related topics, event, etc, and we welcome any contribution to
+it. As our community is highly dynamic, information there can become out of
+date quickly. Feel free to tell us when this is the case, or if anything is
+missing.
+
+
+Weekly Dev meeting
+------------------
+
+The core developer have weekly dev-meeting to request help, or discuss things
+that happen recently, as well as bootstrap conversation. We try to keep the
+meeting short. These meeting are public and posted on the `IPython channel
+<https://www.youtube.com/channel/UCUuzz1eYiKIzu_Uw1ZQLNoQ>`_ on YouTube.
+
+You will find there two kinds of meetings, the ``IPython/Jupyter``  ones
+– usually at 9am Pacific time on Tuesday – that are broad topic, and the
+``JupyterLab`` ones – usually 9am Pacific Time on Friday –  which are more
+focused on the current evolution of the Jupyter Notebook. These meeting are not
+live broadcast for technical reasons, but we are always looking for a better
+technologies, so if you have any advice, please let us know.
+
+Collaborative notes are taken before/during the meeting on
+https://jupyter.hackpad.com  [a real-time markdown-like editor] which contain
+the daily link to the video meeting (if you want to join live), but the
+attendance is limited to 10 people, after which the video conferenced becomes
```

Comment:
we don't really limit attendance to 10, and BJ officially supports up to 100.  I would just remove everything starting at "but the attendance..."